### ISSUE TITLE
Screen share does not work

### ISSUE THREAD

**Issue by alialic** [bigbluebutton/bbb-install#440]
When connecting to a session via demo, there is no problem sharing the screen,
![demo](https://user-images.githubusercontent.com/65443862/136756048-c2518114-8213-4b86-b259-fe1a49a63519.PNG)

 but when connecting, the screen is not shared via the api.
 
![api](https://user-images.githubusercontent.com/65443862/136756098-8fa74880-1622-4ada-be3e-221a91a60b82.PNG)

This has happened in the last few days

**Comment by alialic** [comment]
Yes ,using the API demos, I was able to successfully start and view screen sharing ,but without API demos ,i can not view screen share. 

> Can you share your screen when connecting to https://demo.bigbluebutton.org/ but not your own site?
> 
> Using the API demos, I was able to successfully start and view screen sharing on your site; however, the version of BigBlueButton is quite old. Recommend updating to latest version of BigBlueButton 2.3.
> 
> Also, the site still has the API demos installed -- you'll want to remove them after testing.

**Comment by ffdixon** [comment]
> Yes ,using the API demos, I was able to successfully start and view screen sharing ,but without API demos ,i can not view screen share.

To confirm, 

1. you can start and view screen share using API demos on your BigBlueButton server.  
2. However, using a different front-end to start a BigBlueButton session, you can not view screen share on your BigBlueButton server.

If (1) is true, then your server is working.  You haven't shared any information about (2), so at best we would recommend looking at the incoming API requests from both (1) and (2) and compare them to see if there are any differences.

You can view the incoming API request by tailing `/var/log/nginx/bigbluebutton.access.log`.
